atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -136,16 +137,25 @@ def run(self):
                 registry_image.registry = registry
                 if self.need_skopeo_push():
                     self.push_with_skopeo(registry_image, insecure, docker_push_secret)
+                    digests = get_manifest_digests(registry_image, registry,
+                                                   insecure, docker_push_secret)
                 else:
-                    self.tasker.tag_and_push_image(self.workflow.builder.image_id,
-                                                   registry_image, insecure=insecure,
-                                                   force=True, dockercfg=docker_push_secret)
+                    for retry in range(DOCKER_MAX_RETRIES):
+                        self.tasker.tag_and_push_image(self.workflow.builder.image_id,
+                                                       registry_image, insecure=insecure,
+                                                       force=True, dockercfg=docker_push_secret)
+                        digests = get_manifest_digests(registry_image, registry,
+                                                       insecure, docker_push_secret)
+                        if (not (digests.v2 or digests.oci) and (retry + 1 != DOCKER_MAX_RETRIES)):
```

Comment:
Should we check if v2 schema 2 is desired based on configuration?

Reply:
yeah I'll check if group_manifest are set and also if v2-2 is expected